spec/std/concurrent_spec.cr

```diff
@@ -66,4 +66,22 @@ describe "concurrent" do
   it "accepts method call with receiver" do
     typeof(spawn String.new)
   end
+
+  it "schedules intermitting sleeps" do
+    chan = Channel(Int32).new
+    spawn do
+      3.times do |i|
+        sleep 40.milliseconds
+        chan.send (i + 1)
+      end
+    end
+    spawn do
+      2.times do |i|
+        sleep 100.milliseconds
+        chan.send (i + 1) * 10
+      end
+    end
+
+    Array(Int32).new(5) { chan.receive }.should eq [1, 2, 10, 3, 20]
```

Comment:
This newly added spec fails on darwin. I've already increased the sleep times trying to avoid inconsistencies.
Could someone run this example on macOS to find a sleep amount that leads to success consistently? Or investigate why if it continues to fail? :D

Reply:
Mmm... We need this in order to get the final approval, otherwise the macOS test will start failing on master.

Reply:
Do we know why it fails? Those times indicate some pretty insane overhead?

Reply:
I don't know why. We need someone to test this on macOS manually.

But I'll disable this spec on darwin for now, to keep CI happy.